src/components/generic/ReactNavbar.tsx

```
@@ -0,0 +1,275 @@
+                  className={`rounded-lg px-4 py-2 text-lg font-medium transition-colors ${
+                    isActive ? activeTextClass : baseTextClass
+                  }`}
+                >
+                  {link.label}
+                  {link.external && (
+                    <svg
+                      className="ml-1 inline-block h-3 w-3 opacity-50"
+                      fill="none"
+                      viewBox="0 0 24 24"
+                      stroke="currentColor"
+                    >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)